mk/aosp_optee.mk

```diff
@@ -50,7 +50,10 @@ BUILD_OPTEE_OS:
 		CFG_ARM64_core=$(OPTEE_CFG_ARM64_CORE) \
 		PLATFORM=$(OPTEE_PLATFORM) \
 		PLATFORM_FLAVOR=$(OPTEE_PLATFORM_FLAVOR) \
-		$(CROSS_COMPILE_LINE)
+		$(CROSS_COMPILE_LINE) \
+		CFG_TEE_CORE_LOG_LEVEL=3 \
```

Comment:
Or can we add something like `$(OPTEE_EXTRA_FLAGS)` here and try to set the log level from one of the aosp .mk files?